pvlib/singlediode.py

```diff
@@ -949,3 +950,85 @@ def _pwr_optfcn(df, loc):
                                  df['resistance_shunt'], df['nNsVth'])
 
     return current * df[loc]
+
+
+def batzelis_keypoints(photocurrent, saturation_current, resistance_series,
```

Comment:
```suggestion
def batzelis(photocurrent, saturation_current, resistance_series,
```

This function has the same signature as `pvsystem.singlediode` so I don't think it helps to have "keypoints" in the name.